### DISCUSSION TITLE
v0.4.5

### DISCUSSION THREAD

**Discussion by hyperfield** [hyperfield/yt-channel-downloader#18] [Announcements]
The latest downloadable installer of the latest release of YT Channel Downloader for Windows (compiled from the updated repository). This installer also includes the uninstaller and a script to download and install ffmpeg on your system if it's not present. ffmpeg is a set of libraries and tools for multimedia processing, and it is needed for YT Channel Downloader to work properly.

[Download the latest installer for Windows here](https://quicknode.net/media/dist/YT_Channel_Downloader_0_4_5_installer.zip)

**Full Changelog**: https://github.com/hyperfield/yt-channel-downloader/compare/v0.4.2...v0.4.5

<hr /><em>This discussion was created from the release <a href='https://github.com/hyperfield/yt-channel-downloader/releases/tag/v0.4.5'>v0.4.5</a>.</em>

**Comment by nellytoxy** [comment]
trying to download and install it but my antivirus keeps detecting virus is the file safe

**Reply by hyperfield** [reply]
There's no virus there. See, for example,

https://github.com/pyinstaller/pyinstaller/issues/6754
https://stackoverflow.com/questions/43777106/program-made-with-pyinstaller-now-seen-as-a-trojan-horse-by-avg
https://coderslegacy.com/pyinstaller-exe-detected-as-virus-solutions/
https://medium.com/@markhank/how-to-stop-your-python-programs-being-seen-as-malware-bfd7eb407a7
https://plainenglish.io/blog/pyinstaller-exe-false-positive-trojan-virus-resolved-b33842bd3184

It's a known issue with `pyinstaller`, the tool I used to create the executable file (essentially a bundle of Python files). Some antivirus software, like Avast, mistakenly detects it as malware, resulting in a false positive.

In this case, instead of using the installer, you can follow the `Readme` on Github (https://github.com/hyperfield/yt-channel-downloader) to just run the native (non-packed) Python version of the code.

I have a plan to write a program in order to solve this issue, but that'll take some time.
